classes/rawemailmessage.class.inc.php

```diff
@@ -172,6 +172,21 @@ public function GetMessageId()
 	 */
 	public function GetAttachments(&$aAttachments = null, $aPart = null, &$index = 1)
 	{
+		// This Regex complies with RFC 2045 regarding the Grammar of Content Type Headers Filenames:
+		// 	(1) Allow all chars for the filename, if the filename is quoted with double quotes
+		//	(2) Allow all chars for the filename, if the filename is quoted with single quotes
+		//	(3) If the filename is not quoted, allow only ASCII chars and exclude the following
+		//			chars from the set, referenced as "tspecials" in the RFC:
+		//					<ALL-CTL-CHARS-INCL-DEL>
+		//					<CHAR-SPACE>
+		//					()<>@,;:\"/[]?=
+		// 			To keep the regex as simple as possible, the _allowed_ chars are whitelisted
```

Comment:
```suggestion
		// 			To keep the regex as simple as possible, the _allowed_ chars are listed
```

We tend not to use "white/black" terms but "allowed/denied" instead, in this case it can be considered redundant.

Reply:
Check.